dolphinscheduler-task-plugin/dolphinscheduler-task-api/src/main/java/org/apache/dolphinscheduler/plugin/task/api/parameters/DependentParameters.java

```diff
@@ -21,11 +21,13 @@
 import org.apache.dolphinscheduler.plugin.task.api.model.DependentTaskModel;
 
 import java.util.List;
+import java.util.Map;
 
 public class DependentParameters extends AbstractParameters {
 
     private List<DependentTaskModel> dependTaskList;
     private DependentRelation relation;
+    private Map<String, Object> otherParams;
```

Comment:
Could you please describe what is otherParams? And why if the dependent is not exist, we still need to set the otherParams? this is strange.

Reply:
This is another parameter, which can be extended later; In addition, if the dependent does not exist, it will not be set